stories/components/researchcontextpage.stories.ts

```diff
@@ -0,0 +1,15 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { ResearchContextPage } from "@/components/researchcontextpage";
+
+const meta: Meta = {
+  title: "Components/researchcontextpage",
```

Comment:
Please use consistent casing in the naming. Following your implicit criteria, this should be "ResearchContextPage"